reporting/src/webrequest-pipeline/page-store.js

```diff
@@ -389,14 +291,6 @@ export default class PageStore {
       return null;
     }
     const page = new Page(serializedPage);
-    // check if the current page has the given frame id, otherwise check if it belongs to the
-    // previous page
-    if (!page.frames[frameId]) {
-      if (page.previous && page.previous.frames[frameId]) {
-        return page.previous;
```

Comment:
Note that this can still be reached. In the example that I have seen, `couldBePreviousPage` would have been false, we will then attribute the request to the current tab. I would expect it to have a negative impact on data quality.